src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -713,7 +713,9 @@ bool createGlobalConfigFiles()
                         {
                             CreateGlobalconfig(
                                 analyzerGlobalconfigsDir,
-                                $"AnalysisLevel_{analysisLevelVersionString}_{analysisMode}.editorconfig",
+#pragma warning disable CA1308 // Normalize strings to uppercase
+                                $"AnalysisLevel_{analysisLevelVersionString}_{analysisMode!.ToString()!.ToLowerInvariant()}.editorconfig",
```

Comment:
Isn't it better to have assertions or null check?

Reply:
I believe the C# compiler is generating nullable warnings because both Enum.GetValues and ToString are annotated as something that can return null, which is obviously not possible here.